recipe/meta.yaml

```diff
@@ -48,7 +45,11 @@ about:
   license_family: MIT
```

Comment:
The license is Apache-2.0, see https://github.com/visgl/deck.gl/blob/master/bindings/pydeck/LICENSE.txt
And `license_family: Apache`

Reply:
wow good catch! fixed